src/components/sidebar/sidebar.stories.js

```diff
@@ -52,6 +55,7 @@ export const Default = () => {
         position={position}
         size={size}
         onCancel={onCancel}
+        header={headerChildren.length > 0 ? headerChildren : null}
```

Comment:
```suggestion
        header={headerChildren}
```

No need for the extra bits 👍🏻

Reply:
Consider it gone.